src/lxc/caps.c

```diff
@@ -247,6 +244,8 @@ bool lxc_file_cap_is_set(const char *path, cap_value_t cap, cap_flag_t flag)
 	cap_is_set = lxc_cap_is_set(caps, cap, flag);
 	cap_free(caps);
 	return cap_is_set;
+	#else
+	return true;
```

Comment:
In fact another way to implement this would be to do

#ifndef LIBCAP_HAS_FILE_CAPS
#define cap_get_flags(a, b, c, d) { errno = ENODATA; return NULL; }
#endif

 and just treat it as a failed helper call.